### ISSUE TITLE
add switch for javax.el-api version

### ISSUE THREAD

**Issue by donsizemore** [IQSS/dataverse-ansible#72]
default to javax.el-api-3.0-b05.jar

**Comment by donsizemore** [comment]
are we there already?

```
vagrant@localhost ~/javax/el $ /usr/local/glassfish4/bin/asadmin osgi lb |grep -i expression
  137|Resolved   |    1|Expression Language 3.0 (3.0.0)
```

**Comment by pdurbin** [comment]
I probably asked for this but I can't even remember why. Closing.
